decidim-core/vendor/assets/javascripts/tribute.js

```diff
@@ -617,8 +617,10 @@ function () {
           }
         }
 
-        tribute.selectItemAtIndex(li.getAttribute('data-index'), event);
-        tribute.hideMenu(); // TODO: should fire with externalTrigger and target is outside of menu
+        if (li.getElementsByClassName("disabled-tribute-element")[0]==undefined){
```

Comment:
As we are changing this file, can we add a comment on the top of the file indicating that this is not the same vendor file that can be found in the tribute website?